workflows/reference_files/2.4/reference_files.smk

```diff
@@ -705,10 +705,11 @@ rule _imgt_db_success:
 
 rule download_oncodrive_refs:
     output:
-        refs = "downloads/oncodrive/datasets/genomereference/{oncodrive_build}.master",
-        stops = "downloads/oncodrive/datasets/genestops/{oncodrive_build}.master"
+        refs = "downloads/oncodrive/{version}/datasets/genomereference/{oncodrive_build}.master",
+        stops = "downloads/oncodrive/{version}/datasets/genestops/{oncodrive_build}.master"
     params:
-        outdir = "downloads/oncodrive/"
+        outdir = "downloads/oncodrive/{version}/",
+        provider = lambda w: config["genome_builds"][w.version]["provider"]
```

Comment:
What is the provider param for? I've seen it used in other reference_files rules but I'm not sure what it does in this rule

Reply:
It will basically determine whether or not the reference file is chr-prefixed or not :smile: